Makefile

```diff
@@ -201,7 +201,7 @@ dist: clean
 	CGO_ENABLED=0 $(GOX) -osarch="linux/amd64 linux/arm64 linux/arm darwin/amd64 windows/amd64 freebsd/amd64" ./cmd/loki-canary
 	CGO_ENABLED=0 $(GOX) -osarch="linux/arm64 linux/arm darwin/amd64 windows/amd64 freebsd/amd64" ./cmd/promtail
 	CGO_ENABLED=1 $(CGO_GOX) -osarch="linux/amd64" ./cmd/promtail
-	gzip dist/*
+	for i in dist/*; do zip -j $$i.zip $$i && rm -f $$i; done
 	pushd dist && sha256sum * > SHA256SUMS && popd
```

Comment:
I'd prefer:

```bash
pushd dist
for i in *; do zip -j -m $$i.zip $$i; done
sha256sum * > SHA256SUMS
popd

Reply:
> I'd prefer:
> 
> ```shell
> pushd dist
> for i in *; do zip -m $$i.zip $$i; done
> sha256sum * > SHA256SUMS
> popd
> ```

For this to work, this target needs to run in "ONESHELL" mode, because pushd/popd are shell built-ins. https://www.gnu.org/software/make/manual/html_node/Execution.html#Execution

Reply:
Ah man, Makefiles killing me

Reply:
so i would not do this change.

Reply:
zip -m instead of rm -rf